### DISCUSSION TITLE
How to use the [image] shortcut inside a HTML div element?

### DISCUSSION THREAD

**Discussion by meentix** [datenstrom/community#95] [Ask a question] [answered]
Hello,

it seems that I can not use the [image]-Tag within a div. For example:

<div>[image picture.jpg Picture left]</div>

That will not execute :-(

Can someone confirm and fix it?

**Comment by nibreh** [comment]
You mean you use this when editing your page online ? 

`<div class="name">[image picture.jpg Picture left]</div>`

**Comment by meentix** [comment]
yes and the image is not shown up, only the code.

**Comment by nibreh** [comment, marked as answer]
OK

Insert markdown in a html tag is not working, this is normal, but you can use this trick:

`<div class="name" markdown="1">[image picture.jpg Picture left]</div>`

just add `markdown="1"`

tell me

**Comment by meentix** [comment]
Perfect, that works :-) Thank you.
